### DISCUSSION TITLE
blog/taiwan-narrative

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#47] [Announcements]
# blog/taiwan-narrative

Two thousand seven hundred and fifty words

https://pathos.page/blog/taiwan-narrative

**Comment by savouer1** [comment]
我有GIT账号了，终于能在你这评论了。——夜未央
从历史经验看，独立与否不在于叙事方式，也不在于法理的独立，决定因素还是在拳头的大小。没有实力将想象变成事实前，所有的叙事都是意淫。

**Reply by Pathsis** [reply]
恭喜😄

尽管一切归根结底是实力的问题，但不是一切都是实力的问题。理论和意识形态工作还是要做的。

**Reply by calon** [reply]
有时候叙事和意识影响了拳头能不能攥紧。
